DTS-STN/Alpha-Site
# Project page UI updates

# Description

[Project page UI updates](https://trello.com/c/3Co8ZYK4)

Project label colours have been changed:
![image](https://user-images.githubusercontent.com/34345435/133641542-77130e91-64df-4a3d-8ba9-d5b8d20dd104.png)

The filter buttons have been updated to accommodate mobile screens, though it's still off. 
![image](https://user-images.githubusercontent.com/34345435/133641712-da31d916-9357-42b3-9ca1-639a15bee63f.png)

## Acceptance Criteria

Project labels have updated colours and the filter bar fits mobile screen too.

## Test Instructions

1. Visit the projects page
2. Notice the new colours for project status
3. Change to mobile view
4. Notice that the filter bar doesn't get put on separate lines

## Help Requested

- The text in the filter bar is not centered vertically (for the "all" option). If anyone knows how to fix that, please help.

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
